portafly/doc/ux_specs/global_components/alerts.md

```diff
@@ -0,0 +1,13 @@
+# Alerts UX/UI specs
+
+[Back to index](../index.md)
+
+* [PatternFly Alerts design guidelines](https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/alerts-and-notifications)
+* [PatternFly-react Alerts docs](https://www.patternfly.org/v4/documentation/react/components/alert)
+
+### Input field validation errors / alerts
+* This type of alerts will be displayed as **inline alerts**
```

Comment:
```suggestion
* These types of alerts will be displayed as **inline alerts**
```

Reply:
Depending on what you're trying to convey here, it could be: `* This type of alert will be displayed as an **inline alert**
`

Reply:
@dfennessy Thank you!